SendEmail/README.md

```diff
@@ -0,0 +1,48 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-communication-services
+---
+
+# Email Sample
+
+This sample application shows how the Azure Communication Services Email SDK can be used to build an email experience. This sample sends an email to the required recipients of any domain using [Email Communication Services resource](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/email/create-email-communication-resource)
```

Comment:
Suggested change: "This sample sends an email to the **selected** recipients of any domain using **an** [Email Communication Services resource]".

Although the "recipients" field is technically required, it is not a requirement that you send an email to the recipients you select.

Reply:
"it is not a requirement that you send an email to the recipients you select." Can you recheck if this is right?

Reply:
Yes, what I said is correct. The API requires that you populate the "recipients" field. Grammatically, however, it is incorrect to say that the sample sends emails to _required_ recipients because the recipients that one user selects to send to may be different than others, and there is no requirement for a _specific_ set of recipients between users of this guide.